### DISCUSSION TITLE
Proposta grafica valutazioni docente

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#648] [Idee e proposte]
Buongiorno,mi sarei preso qualche minuto su word per implementare e forse rendere un po' più concreta l'idea di come inserire la  valutazione senza trascinare ogni volta il cursore .Allego 
<img width="759" alt="Screenshot 2024-02-12 alle 14 42 05" src="https://github.com/iisgiua/giuaschool/assets/93823125/4737f240-be68-43a6-b2e8-09204ac49ab0">

**Comment by gianlucaaiello56** [comment]
Io credo che non sia funzionale inserirlo liberamente, sarebbe meglio un menu a tendina in relazione alla tua proposta ! 
Complimenti per l'idea !

**Comment by sempliciniluigi89** [comment]
Magari la tendina con le valutazioni personalizzabili dalla segreteria che
può aggiungere,modificare o eliminare la tipologia di voto.Poi sarebbe
utile specificare sia al lato docente che genitore/alunno  il voto con il
valore , ad esempio quando si mette un voto es. 9+ "valore (9.25)" , 9-
"valore (8.85/75 in base a come deciderà la segreteria)"

Il giorno mar 13 feb 2024 alle ore 10:37 gianlucaaiello56 <
***@***.***> ha scritto:

**Reply by Davide031** [reply]
Ottimo lavoro, sarebbe molto utile

**Comment by trinko** [comment]
La segno come richiesta. Prima o poi, quando il tempo a disposizione e altre priorità me lo permetteranno, conto di fare una revisione della gestione dei voti, configurabile in modo che sia facilmente adattabile ad ogni esigenza.
